packages/kolibri/urls.js

```diff
@@ -65,19 +65,16 @@ class UrlResolver {
 
     const patterns = this._patterns[name];
     if (!patterns) {
-      urlFunc = () => {
-        if (process.env.NODE_ENV !== 'production') {
-          if (name.includes('-')) {
-            throw new Error(
-              `URL pattern names should use underscores instead of dashes. Try "${name.replace('-', '_')}"`,
-            );
-          }
+      if (process.env.NODE_ENV !== 'production') {
+        if (name.includes('-')) {
+          throw new Error(
+            `URL pattern names should use underscores instead of dashes. Try "${name.replace('-', '_')}"`,
+          );
         }
-        throw new Error(`URL pattern "${name}" not found`);
```

Comment:
So this has been deleted because we don't want to throw an error here if not found, we want to just advance to `urlFunc = this._createUrlFunction(name, patterns);` ?